modules/porous_flow/doc/tests/mandel.py

```diff
@@ -0,0 +1,167 @@
+    fluid_mobility = 1.5
+    normal_stress = 1.0
+
+    # derived parameters
+    soil_shear_modulus = soil_lame_mu
+    soil_drained_bulk = soil_lame_lambda + 2.0 * soil_lame_mu / 3.0
+    fluid_bulk_compliance = 1.0 / fluid_bulk_modulus
+    biot_modulus = 1.0 / (initial_porosity / fluid_bulk_modulus + (biot_coeff - initial_porosity) * (1.0 - biot_coeff) / soil_drained_bulk)
+    undrained_bulk_modulus = soil_drained_bulk + biot_coeff**2 * biot_modulus
+    skempton = biot_coeff * biot_modulus / undrained_bulk_modulus
+    drained_poisson = (3.0 * soil_drained_bulk - 2.0 * soil_shear_modulus) / (6.0 * soil_drained_bulk + 2.0 * soil_shear_modulus)
+    undrained_poisson = (3.0 * undrained_bulk_modulus - 2.0 * soil_shear_modulus) / (6.0 * undrained_bulk_modulus + 2.0 * soil_shear_modulus)
+    consolidation_coeff = 2.0 * fluid_mobility * skempton**2 * soil_shear_modulus * (1.0 - drained_poisson) * (1 + undrained_poisson)**2 / 9.0 / (1.0 - undrained_poisson) / (undrained_poisson - drained_poisson)
+
+    roots = [1.419988120304100E+00, 4.666177581823210E+00, 7.826417353528760E+00, 1.097591703059930E+01, 1.412188800507350E+01, 1.726626279765500E+01, 2.040978005325610E+01, 2.355278342938330E+01, 2.669545454962390E+01, 2.983789845132980E+01, 3.298018011077390E+01, 3.612234188229790E+01]
+    expr1 = [np.sin(v) / (v - np.sin(v) * np.cos(v)) for v in roots]
+    expr2 = [np.sin(v) * np.cos(v) / (v - np.sin(v) * np.cos(v)) for v in roots]
+
+
+    d_terms = [expr2[i] * np.exp(- roots[i]**2 * consolidation_coeff * t / soil_width**2) for i in range(len(roots))]
+    # following is for t = 0
+    # vert_disp = - normal_stress * soil_height * (1.0 - undrained_poisson) / 2.0 / soil_shear_modulus / soil_width
+    # hor_disp = normal_stress * undrained_poisson / 2.0 / soil_shear_modulus
+    vert_disp = - normal_stress * (1.0 - drained_poisson) * soil_height / 2.0 / soil_shear_modulus / soil_width + normal_stress * (1.0 - undrained_poisson) * soil_height / soil_shear_modulus / soil_width * sum(d_terms)
+    hor_disp = normal_stress * drained_poisson / 2.0 / soil_shear_modulus + normal_stress * (1.0 - undrained_poisson) / soil_shear_modulus * sum(d_terms)
+
+    p_terms = [(expr1[i] * np.cos(roots[i] * x / soil_width) - expr2[i]) * np.exp(- (roots[i] / soil_width)**2 * consolidation_coeff * t) for i in range(len(roots))]
+    porepressure = 2.0 * normal_stress * skempton * (1.0 + undrained_poisson) / 3.0 / soil_width * sum(p_terms)
+
+    return (vert_disp, hor_disp, porepressure)
+
+def get_moose_results(fi):
+    f = open(fi)
+    data = [map(float, line.strip().split(",")) for line in f.readlines()[1:] if line.strip()]
+    f.close()
+    t = [d[0] for d in data]
+    p0 = [d[1] for d in data]
+    p3 = [d[4] for d in data]
+    p8 = [d[9] for d in data]
+    force = [d[12] for d in data]
+    xdisp = [d[13] for d in data]
+    ydisp = [d[14] for d in data]
+    return (t, p0, p3, p8, force, xdisp, ydisp)
+
+tpoints = np.arange(1E-5, 0.71, 1E-3)
```

Comment:
OK, thanks for the tip-off.  I'll keep this in mind.  My python scripts are pretty hacky!